pkgs/jnigen/example/kotlin_plugin/lib/kotlin_bindings.dart

```diff
@@ -41,11 +41,11 @@ class Example extends jni.JObject {
 
   /// The type which includes information such as the signature of this class.
   static const type = $ExampleType();
-  static final _id_new0 = _class.constructorId(
```

Comment:
This does not look consistent with the description above, it's not `$`+digits?

Reply:
As it's private, I didn't add a special case for it.